gammapy/astro/darkmatter/tests/test_spectra.py

```diff
@@ -23,11 +23,11 @@ def test_primary_flux():
 
 
 @pytest.mark.parametrize(
-    "mass, expected_flux", [(1.6, 0.00025037), (11, 0.00502079), (75, 0.02028309)]
+    "mass, expected_flux, source", [(1.6, 0.00025037,'pppc4'), (11, 0.00549445,'cosmixs'), (75, 0.02028309,None)]
```

Comment:
`None` defaults to pppc4, see comment in spectra.py. Maybe one rather wants to test a `source="nonexistend"` with xfail